<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsExtent3DWidget</class>
 <widget class="QWidget" name="QgsExtent3DWidget">
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string notr="true">Extent</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout" stretch="0,0,0,0">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="sizeConstraint">
    <enum>QLayout::SetMinimumSize</enum>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="calculateLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <item>
      <widget class="QLabel" name="calculateLabel">
       <property name="text">
        <string>Set extent from:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="mCalculateButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>…</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="calculateSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="infoHorizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <property name="alignment">
           <set>Qt::AlignTop</set>
         </property>
     <item>
       <widget class="QTableWidget" name="mCoordinateTable">
         <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
           </sizepolicy>
         </property>
         <property name="sizeAdjustPolicy">
           <enum>QAbstractScrollArea::AdjustToContents</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
       <property name="rowCount">
        <number>2</number>
       </property>
       <property name="columnCount">
        <number>2</number>
       </property>
       <row>
        <property name="text">
         <string>X</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>Y</string>
        </property>
       </row>
       <column>
        <property name="text">
         <string>Min</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Max</string>
        </property>
       </column>
       <item row="0" column="0">
        <property name="toolTip">
         <string>South</string>
        </property>
       </item>
       <item row="0" column="1">
        <property name="toolTip">
         <string>North</string>
        </property>
       </item>
       <item row="1" column="0">
        <property name="toolTip">
         <string>West</string>
        </property>
       </item>
       <item row="1" column="1">
        <property name="toolTip">
         <string>East</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
       <layout class="QFormLayout" name="infoFormLayout">
        <property name="labelAlignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       <item row="0" column="0">
        <widget class="QLabel" name="mCrsLabel">
         <property name="text">
          <string>Used CRS:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="mCrsValue">
         <property name="text">
          <string>…</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="mLengthLabel">
         <property name="text">
           <string>Extent length (m):</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="mLengthValue">
         <property name="text">
          <string>…</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="mWidthLabel">
         <property name="text">
           <string>Extend width (m):</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="mWidthValue">
         <property name="text">
          <string>…</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="mSurfaceLabel">
         <property name="text">
           <string>Extent surface (m²):</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="mSurfaceValue">
         <property name="text">
          <string>…</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="mPreviewExtentCheckBox">
     <property name="text">
      <string>Show in 2D map view</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
